src/backend/joanie/core/api/client.py

```diff
@@ -45,13 +45,23 @@ def get_queryset(self):
         """
         queryset = super().get_queryset()
         course_id = self.kwargs.get("course_id")
+        search_filter = self.request.query_params.get("search", None)
```

Comment:
As per this morning's discussion :wink: 
```suggestion
        search_filter = self.request.query_params.get("query", None)
```